apps/pie-microsite/src/tokenCategories.json

```diff
@@ -175,5 +175,28 @@
     }
   },
   "radius": {},
-  "spacing": {}
+  "spacing": {},
+  "elevation": {
+      "global": {},
+      "alias": {
+        "default": {
+          "dark": {
```

Comment:
I don't think you need to split these by light and dark. Can we do it the same way colour tokens does?

Reply:
I think in this instance it needs to replicate font and use the path to fetch the categories. Happy to discuss further though. If I remove the dark and light, `regularCategories` becomes undefined and it sends me to `buildUncategorisedLists` 😄 

https://github.com/justeattakeaway/pie/pull/160/files#diff-5b6a7bfb4cc7549cd82f001c49a5168193f934bb0b48e1304ecd0351c81a3bcbL292

Reply:
I've had a look this morn and it does seem to be just a case of refactoring the logic. If we want it to be similar to `parentCategories` we'd need to adjust the metadata.json file. Thinking I'll have a go at this in a separate PR - this one's already getting quite lengthy 👍